### PR TITLE
allow to skip not segmented subjects

### PR DESCRIPTION
**Pull request by ntraut**
related to issue #12
I also added a skip for the resample into fsaverage4 and the creation of volumes tables if the files already exist.

### CODE REVIEW

**Comment by fliem** [comment]
Thanks.
Could you please also add some tests in .circleci/config.yml

**Comment by fliem** [comment]
Thanks @ntraut. Circleci was configured not to run tests on forks, that's why your PR didn't trigger the CI. I have changed the settings. Could you please trigger the CI by making another commit (e.g., add a comment or a new line). Thanks.

File: baracus/utils.py [hunk 1]
```diff
@@ -32,21 +32,28 @@ def run_fs_if_not_available(bids_dir, freesurfer_dir, subject_label, license_key
...
+                                                                 n_cpus=n_cpus)
 
-            print("Freesurfer for {} not found. Running recon-all: {}".format(subject_label, cmd))
-            run(cmd)
+        print("Freesurfer for {} not found. Running recon-all: {}".format(subject_label, cmd))
+        run(cmd)
+
+    for fss in freesurfer_subjects:
+        if not os.path.isfile(os.path.join(freesurfer_dir, fss, "stats/aseg.stats")):
+            freesurfer_subjects.remove(fss)
```
Comment (fliem) [inline baracus/utils.py]: Why is this necessary?

File: scripts/run_brain_age_bids.py [hunk 2]
```diff
@@ -86,7 +87,8 @@
         df = pd.DataFrame([])
         for subject in freesurfer_subjects_to_analyze:
             in_file = os.path.join(out_dir, subject, subject + "_predicted_age.tsv")
-            df = df.append(pd.read_csv(in_file, sep="\t"))
+            if os.path.isfile(in_file):
+                df = df.append(pd.read_csv(in_file, sep="\t"))
```
Comment (fliem) [inline scripts/run_brain_age_bids.py:91]: Could you please make this raise an error if `--skip_missing` is not set and the file that should be there is missing:

```
            if os.path.isfile(in_file):
                df = df.append(pd.read_csv(in_file, sep="\t"))
            elif not args.skip_missing:
                raise FileNotFoundError(in_file)
```

File: baracus/utils.py (hunk 1 above)
Comment (ntraut) [reply baracus/utils.py]: It is because sometimes segmentation fails so in this case you need to ignore the subject.

Comment (fliem) [reply baracus/utils.py]: I see. Could you please also raise an exception if skip_missing is not set and the file is missing.

**Comment by fliem** [comment]
Merging. Thanks for your contribution.
